src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -147,10 +149,44 @@ private ContextMenu createContextMenu(){
         final ContextMenu contextMenu = new ContextMenu();
 
         MenuItem editMenuItem = new MenuItem("Edit");
+        editMenuItem.setAccelerator(ShortcutsManager.SHORTCUT_PERSON_EDIT);
         editMenuItem.setOnAction(e -> handleEditPerson());
         MenuItem deleteMenuItem = new MenuItem("Delete");
+        deleteMenuItem.setAccelerator(ShortcutsManager.SHORTCUT_PERSON_DELETE);
         deleteMenuItem.setOnAction(e -> handleDeletePerson());
         contextMenu.getItems().addAll(editMenuItem, deleteMenuItem);
         return contextMenu;
     }
+
+    @Subscribe
+    private void HandleDeleteRequestEvent(DeleteRequestEvent event) {
```

Comment:
This and `HandleEditRequestEvent` are unused. Should be named `handleDeleteRequestEvent` (no capitalization of `H`).